### PR TITLE
Hair 125 remove chakra materialui

### PR DESCRIPTION
**Pull request by cynthiaeddy**
## Description

Removed dependencies: chakra, material/ui and emotion.
Refactored code in user registration, edit profile and add business where material/ui icons (replaced with react icons) and material/ui core were involved.

## Related Issue

closes #36

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|    | :sparkles: New feature     |
|   ✓  | :hammer: Refactoring       |
|    | :100: Add tests            |
|   ✓  | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

|   before  | after                       |
| --- | -------------------------- |
| Screen Shot 2022-05-23 at 7 34 12 PM | Screen Shot 2022-05-23 at 7 34 56 PM
   |

### CODE REVIEW

File: client/src/pages/AddBusiness/AddBusiness.tsx [hunk 1]
```diff
@@ -247,20 +250,19 @@ export const AddBusiness = () => {
                     height='100px'
                   />
                   <label htmlFor='select-image'>
-                    <Button
```
Comment (CodeMeKathy) [inline client/src/pages/AddBusiness/AddBusiness.tsx:250]: Hi @cynthiaeddy! Per our conversation, currently, without the `Button` from '@material-ui/core/Button,' the upload image new widow popover functionality breaks. I think we should find an alternative working solution before removing this dependency.

**Review by CodeMeKathy — CHANGES_REQUESTED** [review]

File: client/src/pages/AddBusiness/AddBusiness.tsx (hunk 1 above)
Comment (cynthiaeddy) [reply client/src/pages/AddBusiness/AddBusiness.tsx:246]: hi @CodeMeKathy! I've updated the AddBusiness component to mirror the UserRegistration component as far as adding an image in order to remove material/ui.

**Review by sergiobtos — APPROVED** [review]

**Review by sergiobtos — APPROVED** [review]

**Review by CodeMeKathy — CHANGES_REQUESTED** [review]
HI @cynthiaeddy! Thank you for all your efforts!  Within the `<AddBusiness/>` page changes, the placeholder image is now no longer being stored within the database which makes sense. 
 However, now any business with an empty string for image display a broken image from the `<Profile/>` page within `<MyBusinessList/>`.  See pics.  

<details>
<summary>Images</summary>

image

image

image

image

</details>

**Comment by cynthiaeddy** [comment]
hey @CodeMeKathy business placeholder image is now shown once an owner adds a business in <MyBusinessList> within the <Profile> page. It also appears in <BusinessList>. 

## Updates

|   MyBusinessList   | BusinessList                    |
| --- | -------------------------- |
|Screen Shot 2022-05-30 at 1 28 07 PM|Screen Shot 2022-05-30 at 1 27 24 PM|

**Pull request merged**